src/components/ec-radio-btn/ec-radio-btn.story.ts

```diff
@@ -0,0 +1,91 @@
+import { ref } from 'vue';
+
+import EcRadioBtn from './ec-radio-btn.vue';
+
+export default {
+  title: 'Radio Button',
+  component: EcRadioBtn,
+};
+
+const basicArgs = {
+  options: [
+    { value: 'y', label: 'Yes' },
+    { value: 'n', label: 'No' },
+  ],
+};
+
+export const basic = () => ({
+  components: { EcRadioBtn },
+  setup() {
+    const model = ref('');
+    return {
+      args: basicArgs,
+      label: 'Select one option',
+      model,
+    };
```

Comment:
```suggestion
    const model = ref<RadioBtnProps['modelValue']>('');
    return {
      ecProps: {
        ...basicArgs,
        label: 'Select one option',
        model,
      } as RadioBtnProps,
    };
```

Reply:
i added the 'ref<RadioBtnProps['modelValue']>('');' part, 
typing the returning values is causing me problems

Reply:
this is causing me problems